src/pages/send-tokens/components/send-form-inner.tsx

```diff
@@ -9,10 +9,10 @@ import { useNextTxNonce } from '@common/hooks/account/use-next-tx-nonce';
 import { useSelectedAsset } from '@common/hooks/use-selected-asset';
 import { isEmpty } from '@common/utils';
 import { stacksValue } from '@common/stacks-utils';
-import { TransactionFormValues } from '@common/transactions/transaction-utils';
+import { isTxSponsored, TransactionFormValues } from '@common/transactions/transaction-utils';
 import { ErrorLabel } from '@components/error-label';
 import { ShowEditNonceAction } from '@components/show-edit-nonce';
-import { FeeRow } from '@features/fee-row/fee-row';
+import { FeeRow } from '@components/fee-row/fee-row';
```

Comment:
I did debate this ...I have a hard time knowing what is a feature vs a component. 🤔 To me, I was implementing a new feature.

Reply:
Feature = a logic heavy, data manipulating, possibly—but not necessarily—a reusable component reflecting a significant app feature

Component = a widely reused, environment-agnostic component working only with inputs (props) outputs (events)

Reply:
There's no exact answer, but here (esp after making it dumb) I'd consider it a big component, rather than a small feature.

Reply:
👍